apps/nuxt3-ssr/components/CohortDetails.vue

```diff
@@ -176,6 +186,15 @@ function collectionEventMapper(item: any) {
   };
 }
 
+function datasetMapper(item: any) {
```

Comment:
any ? item struc is known from query

Reply:
same holds for collectionEvent so I just copies that => would be good next step to seperate out those types so we can use them here.